src/ImageSharp/Primitives/Matrix5x4.cs

```diff
@@ -0,0 +1,459 @@
+        /// </summary>
+        public float M54;
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="Matrix5x4"/> struct.
+        /// </summary>
+        /// <param name="m11">The value at row 1, column 1 of the matrix.</param>
+        /// <param name="m12">The value at row 1, column 2 of the matrix.</param>
+        /// <param name="m13">The value at row 1, column 3 of the matrix.</param>
+        /// <param name="m14">The value at row 1, column 4 of the matrix.</param>
+        /// <param name="m21">The value at row 2, column 1 of the matrix.</param>
+        /// <param name="m22">The value at row 2, column 2 of the matrix.</param>
+        /// <param name="m23">The value at row 2, column 3 of the matrix.</param>
+        /// <param name="m24">The value at row 2, column 4 of the matrix.</param>
+        /// <param name="m31">The value at row 3, column 1 of the matrix.</param>
+        /// <param name="m32">The value at row 3, column 2 of the matrix.</param>
+        /// <param name="m33">The value at row 3, column 3 of the matrix.</param>
+        /// <param name="m34">The value at row 3, column 4 of the matrix.</param>
+        /// <param name="m41">The value at row 4, column 1 of the matrix.</param>
+        /// <param name="m42">The value at row 4, column 2 of the matrix.</param>
+        /// <param name="m43">The value at row 4, column 3 of the matrix.</param>
+        /// <param name="m44">The value at row 4, column 4 of the matrix.</param>
+        /// <param name="m51">The value at row 5, column 1 of the matrix.</param>
+        /// <param name="m52">The value at row 5, column 2 of the matrix.</param>
+        /// <param name="m53">The value at row 5, column 3 of the matrix.</param>
+        /// <param name="m54">The value at row 5, column 4 of the matrix.</param>
+        public Matrix5x4(float m11, float m12, float m13, float m14,
+                         float m21, float m22, float m23, float m24,
+                         float m31, float m32, float m33, float m34,
+                         float m41, float m42, float m43, float m44,
+                         float m51, float m52, float m53, float m54)
+        {
+            this.M11 = m11;
+            this.M12 = m12;
+            this.M13 = m13;
+            this.M14 = m14;
+
+            this.M21 = m21;
+            this.M22 = m22;
+            this.M23 = m23;
+            this.M24 = m24;
+
+            this.M31 = m31;
+            this.M32 = m32;
+            this.M33 = m33;
+            this.M34 = m34;
+
+            this.M41 = m41;
+            this.M42 = m42;
+            this.M43 = m43;
+            this.M44 = m44;
+
+            this.M51 = m51;
+            this.M52 = m52;
+            this.M53 = m53;
+            this.M54 = m54;
+        }
+
+        /// <summary>
+        /// Gets the multiplicative identity matrix.
+        /// </summary>
+        public static Matrix5x4 Identity { get; } =
```

Comment:
The term identity matrix [is only valid for square matrices](https://en.wikipedia.org/wiki/Identity_matrix)! We should drop this.

Reply:
See below